tests/test_repository_comments.py

```diff
@@ -30,3 +33,67 @@ async def test_create_comment_success(self):
         
         self.assertEqual(result.created_by, self.user.id)
         self.db.add.assert_called_once_with(result)
+
+    async def test_get_comments_valid_list(self):
+        expected_comments: list[PhotoComment] = [
+                        PhotoComment(
+                id=1,
```

Comment:
We agreed to not hardcode values, related to all tests.

Reply:
done